readme.md

```diff
@@ -0,0 +1 @@
+# datahog
```

Comment:
Can you flesh this out a bit? I don't see this particular readme as being any better than *no* readme -- so far :)